chess/pgn.py

```diff
@@ -1039,7 +1039,7 @@ def read_game(handle, *, Visitor=GameModelCreator):
             elif token in ["1-0", "0-1", "1/2-1/2", "*"] and len(board_stack) == 1:
                 visitor.visit_result(token)
             else:
-                # Replace zeros castling notation.
+                # Replace zeros with correct castling notation.
```

Comment:
Correct is subjective; the PGN specification uses the letter O whereas the FIDE handbook uses the digit 0.

Reply:
Fixed that line. Thanks for the suggestion.